### ISSUE TITLE
[feat]: Achievements Tab

### ISSUE THREAD

**Issue by elvece** [Start9Labs/embassy-os#1646]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already suggest this feature, without success.

### Describe the Feature Request

BE: Implement an endpoint that returns a list of possible achievements, indicating which ones the user has unlocked.  

FE: Create a new tab that displays the unlocked achievements with a fun image / illustration.

TODO:

- [ ] Determine the list of achievements.
- [ ] Relay new endpoint response @dr-bonez 

### Describe the Use Case

Display information about achievements to the user as a fun gamification of eOS.

**Comment by kn0wmad** [comment]
Some suggestions (Achievement label - how to achieve):

- Cherry Pop - Install your first service
- SysAdmin Lvl 1 - Install a service that is dependent on another
- Homebody - Setup LAN
- Lift the Hood - Add an SSH key
- Node Runner - Sync Bitcoin 100%
- Dark Markets - Change to any alternate marketplace
- Tinkerer - Make any 2 settings edits in the Embassy tab
- Researcher Lvl 1 - Click any docs link
- Researcher Lvl 2 - Click 5 docs links
- Helping Hand - Click any support link
- Debugger Lvl 1 - Click into any logs page
- Debugger Lvl 2 - Click into 5 logs pages
- ....this could potentially go on for a while
- Killer - Kill an active session
- Safety First - Successfully create a backup
- Adventurer Lvl 1 - Unlock a hidden menu or menu item
- Adventurer Lvl 2 - Unlock more hidden items
- Housekeeping - Reorder your services

Not sure what all the OS is aware of in these regards or how much of a lift they may be.  I'm happy to help create content for this feature - feel free to assign where appropriate

**Comment by MattDHill** [comment]
Backups! You can also lose this achievement every 4 weeks 😂

**Comment by k0gen** [comment]
Also Recovering from backup should be scored high.
Rescue and Retrieval - Restore one or more services from a prior backup
or
In Recovery 😂

**Comment by kn0wmad** [comment]
Dur... adding to the list

**Comment by chrisguida** [comment]
Might be cool to have a progress bar indicating how many of the important things have been completed...

"Your Embassy is 86% sovereign", etc xD

**Comment by kn0wmad** [comment]
> Might be cool to have a progress bar indicating how many of the important things have been completed...
> 
> "Your Embassy is 86% sovereign", etc xD

Definitely, or leveling system.  Plenty of room to expand after getting the basic functionality
